web/conf.py

```diff
@@ -1214,6 +1214,11 @@
 # it appears on the navigation bar:
 #
 # SEARCH_FORM = """
+SEARCH_FORM = """
+<form class="navbar-search pull-right" action="/search/" role="search">
+    <input type="text" class="form-control" id="tipue_search_input" name="q" placeholder="Search" autocomplete="off">
+</form>
+"""
```

Comment:
I mostly used https://github.com/mate-desktop/mate-desktop.org/blob/8552877b8cdfc8ad28b9e4c4b67f0de10fc7e4dc/conf.py#L1355-L1363 to throw this together.